evm_loader/program/src/config.rs

```diff
@@ -13,7 +13,9 @@ cfg_if! {
         net_specific_config_parser!("config/devnet.toml");
     } else if #[cfg(feature = "govertest")] {
         net_specific_config_parser!("config/govertest.toml");
-    } else {
+    }
+    // Rollup does not need its own config yet, so let it slide to the default one.
```

Comment:
It does need its own config.

@gigimon 
Ask devops to provide operators keys for rollup whitelist. And we need separate chainId and separate Neon token mint.

Reply:
Thanks, I got the data from Dmitry. Added a separate cfg, please check.